src/terminal/parser/ut_parser/OutputEngineTest.cpp

```diff
@@ -1441,6 +1443,13 @@ class StatefulDispatch final : public TermDispatch
         return true;
     }
 
+    bool SetCursorColor(const DWORD color) noexcept override
```

Comment:
I don't know why this was not added before. I need to test this so I add one for my own selfish purpose.